_specification-v1/manifest.md

```diff
@@ -84,9 +80,9 @@ Defines the type of Origami project that the manifest belongs to. **Must** be se
 	</tr>
 </table>
 
-**Must** be set to `1`. It is the version of Origami to which the project conforms.
+**Must** be set to `2`. It is the version of Origami to which the project conforms.
 <pre><code class="o-syntax-highlight--json">{
-	"origamiVersion": 1
+	"origamiVersion": 2
```

Comment:
This is meant to be 2.0 as was agreed here --  https://github.com/Financial-Times/origami/pull/86#issuecomment-712775430

Reply:
Resolved in [`3100293` (#273)](https://github.com/Financial-Times/origami-website/pull/273/commits/3100293cbe93b02e094e45f51ad3cdbe7fdab5cb)